docs/api/spaces-management/post.asciidoc

```diff
@@ -33,6 +33,10 @@ experimental[This functionality is *experimental* and may be changed or removed
 
 `color`:: 
   (Optional, string) Specifies the hexadecimal color code used in the space avatar. By default, the color is automatically generated from the space name.
+
+`imageUrl`:: 
+  (Optional, string) Specifies the data-url encoded image to display in the space avatar. If specified, `initials` will not be displayed, and the `color` will be visible as the background color for transparent images.
```

Comment:
`url` should be `URL`

I've been changing every `If specified` to `When specified`, but totally up to you if it makes sense to change here.

To make the second sentence active, I would change it to "...`initials` are not displayed, and the `color` is visible as the background color for transparent images."